lib/puppet_x/puppetlabs/iis/powershell_version.rb

```diff
@@ -16,18 +16,25 @@ class PowerShellVersion
   module PuppetX
     module PuppetLabs
       module IIS
-        # PowerShellVersion
+        # PowerShell Version
         class PowerShellVersion
```

Comment:
As these constants display in the HTML docs, it may be worth adding a little more info on what they are.

Reply:
hey,

No, these pieces of info are not displayed in the docs

Reply:
If you take a look at `doc/Puppet/Provider/IIS_PowerShell.html` the constants are displayed. Adding something like https://github.com/puppetlabs/puppetlabs-dsc_lite/pull/116/files#diff-c07f1e7257b23123d2709b0192e1ce5fR23-R32 should be enough info for these.